File: template.cpp
#include "rclcpp/rclcpp.hpp"

class MyCustomNode : public rclcpp::Node // You will modify the name
{
public:
    MyCustomNode() : Node("node_name") // You will modify the name
    {
        
    }

private:
};

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);

    auto node = std::make_shared<MyCustomNode>("cpp_test"); // You will modify the name

    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;
}

File: template.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node

class MyCustomeNode(Node): # You will need to modify the name

    def __init__(self):
        super().__init__("node_name") # You will need to modify the name

def main(args=None):
    rclpy.init(args=args)
    node = MyCustomeNode()
    rclpy.spin(node)
    rclpy.shutdown

if __name__ == "__main__":
    main()